### PR TITLE
feat: add 0conf setting

### PR DESCRIPTION
**Pull request by dni**
wip

### CODE REVIEW

File: src/components/settings/ZeroconfSetting.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { useGlobalContext } from "../../context/Global";
+
+const ZeroconfSetting = () => {
```
Comment (michael1011) [inline src/components/settings/ZeroconfSetting.tsx:3]: This is the fourth true/false switch. Let's unify those with one base component

File: src/i18n/i18n.ts [hunk 2]
```diff
@@ -204,6 +204,10 @@ const dict = {
             "Disables prompts to download refund file and other confirmation steps",
         error_starting_qr_scanner:
             "Couldn't access camera, please check permissions!",
+        zeroconf: "Zeroconf",
+        zeroconf_tooltip: "Enable or disable zeroconf",
+        enable_zeroconf: "Enable Zeroconf",
+        enable_zeroconf_tooltip: "Enable or disable zeroconf larger desc",
```
Comment (michael1011) [inline src/i18n/i18n.ts:210]: ?

File: src/i18n/i18n.ts [hunk 3]
```diff
@@ -204,6 +204,10 @@ const dict = {
             "Disables prompts to download refund file and other confirmation steps",
         error_starting_qr_scanner:
             "Couldn't access camera, please check permissions!",
+        zeroconf: "Zeroconf",
```
Comment (michael1011) [inline src/i18n/i18n.ts:207]: Call it `0-conf` in labels

**Review by michael1011 — CHANGES_REQUESTED** [review]
Where is the implementation of that feature?

**Comment by dni** [comment]
still wip, i only through bare bones out there

**Pull request closed without merging**